hayeon/programmers-1/2016년.js

```diff
@@ -0,0 +1,19 @@
+function solution(a, b) {
+  const days = ["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"];
+
+  const date = new Date();
+  date.setFullYear(2016, a - 1, b);
+
+  const result = days[date.getDay()];
+
+  return result;
+}
```

Comment:
```suggestion
function solution(a, b) {
  const days = ["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"];
  const date = new Date(2016, a - 1, b);
  
  return days[date.getDay()];
}
```
- 잘푸셨습니다 !! 메서드 체이닝과 setFullYear을 호출하기 보다 new Date를 통해 인스턴스를 생성할 때 인자로 넘겨서 설정하는 부분을 수정해서 코드를 줄여봤어요~ 참고해주세요 👍

Reply:
훨씬 간단해지네요!!!!! 감사합니닷 ㅎㅎ 🙆🏻‍♀️